### PR TITLE
build(initial): add bit.dev and main dependencies for dev

### PR DESCRIPTION
**Pull request by adhaniscuber**

### CODE REVIEW

File: stackme-kit/core/core.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BasicCore } from './core.composition';
+
+it('should render with the correct text', () => {
+  const { getByText } = render(<BasicCore />);
+  const rendered = getByText('hello from Core');
+  expect(rendered).toBeTruthy();
+});
```
Comment (adityarahmayadi) [inline stackme-kit/core/core.spec.tsx:9]: wrap this one with describe for the better test suite
`describe('Basic core component unit test'){
--- your code right here ---
}
`

File: stackme-kit/core/core.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export type CoreProps = {
+  /**
+   * a text to be rendered in the component.
+   */
+  text: string
+};
```
Comment (adityarahmayadi) [inline stackme-kit/core/core.tsx]: can we move this type definition to separated file? types.ts maybe, wdyt?

File: stackme-kit/icon/icon.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export type IconProps = {
+  /**
+   * a text to be rendered in the component.
+   */
+  text: string
+};
```
Comment (adityarahmayadi) [inline stackme-kit/icon/icon.tsx]: here too, move this one to separated file

File: stackme-kit/ui/button-grid/button-grid.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export type ButtonGridProps = {
+  /**
+   * a text to be rendered in the component.
+   */
+  text: string
+};
```
Comment (adityarahmayadi) [inline stackme-kit/ui/button-grid/button-grid.tsx]: here also move to separated file

File: stackme-kit/ui/button/button.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BasicButton } from './button.composition';
+
+it('should render with the correct text', () => {
+  const { getByText } = render(<BasicButton />);
+  const rendered = getByText('hello from Button');
+  expect(rendered).toBeTruthy();
+});
```
Comment (adityarahmayadi) [inline stackme-kit/ui/button/button.spec.tsx]:
```suggestion
descibe('Basic button unit test', () => {
  it('should render with the correct text', () => {
    const { getByText } = render(<BasicButton />);
    const rendered = getByText('hello from Button');
    expect(rendered).toBeTruthy();
  });
})
```

wdyt?

File: stackme-kit/ui/button/button.tsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export type ButtonProps = {
+  /**
+   * a text to be rendered in the component.
+   */
+  text: string
+};
```
Comment (adityarahmayadi) [inline stackme-kit/ui/button/button.tsx]: move this one to types.ts

**Review by adityarahmayadi — CHANGES_REQUESTED** [review]

File: stackme-kit/ui/button/button.tsx (hunk 6 above)
Comment (adhaniscuber) [reply stackme-kit/ui/button/button.tsx]: sure, we can move to types.ts but inside package UI components, not global

**Pull request closed without merging**